src/ingestor.ts

```diff
@@ -21,7 +21,7 @@ const NETWORK_METADATA = {
     name: 'snapshot',
     version: '0.1.4',
     broviderUrl: process.env.BROVIDER_URL ?? 'https://rpc.snapshot.org',
-    defaultNetwork: '1'
+    defaultNetwork: process.env.NETWORK === 'testnet' ? '11155111' : '1'
```

Comment:
How about using `process.env.DEFAULT_NETWORK ?? '1'` ?

Reply:
ah forgot about this var 😅

Reply:
```suggestion
    defaultNetwork: process.env.DEFAULT_NETWORK ?? '1'
```